### PR TITLE
[DASH1-135] More adjustments to help modal text

### PR DESCRIPTION
**Pull request by stephenyeargin**
Additional edits related to participant status. Carries forward some of the work on #518.

[[DASH1-135](https://precisionmedicineinitiative.atlassian.net/browse/DASH1-135)]

### CODE REVIEW

File: views/dashboard/partials/modal-dashboard-help.html.twig [hunk 1]
```diff
@@ -9,18 +9,19 @@
...
                 </ul>
               </li>
               <li>
-                <strong>Core Participant:</strong> A person who completed the required PPI modules and provided physical measurements and biospecimens. A core participant can contribute to other activities such as providing digital health data using sensor modules and/or wearables, if available.
+                <strong>Core Participant:</strong> Any Fully Consented person who completes the required PPI modules and provides physical measurements and biospecimens is eligible for Core Participant status. A core participant can contribute to other activities such as providing digital health data using sensor modules and/or wearables, if available.
                 <ul>
-                  <li><em>Note:</em> Core Participant status requires receipt of sample for sequencing at the biobank.</li>
+                  <li><em>Note 1:</em> Core Participant status requires receipt of sample for sequencing at the biobank.</li>
+                  <li><em>Note 2:</em> If a Core Participant changes the response to share EHR data at any point, the status will revert to <strong>Participant</strong>.</em></li>
```
Comment (shyamnaru) [inline views/dashboard/partials/modal-dashboard-help.html.twig]: Additional `</em>` tag before `</li>`

File: views/dashboard/partials/modal-dashboard-help.html.twig [hunk 2]
```diff
@@ -9,18 +9,19 @@
...
+              <li><strong>Registered:</strong> A person who created an account by entering a name and either a mobile phone number or an email address and has chosen a language preference but has not yet completed the informed consent process. A registered participant has a unique AoURP participant identifier (PID). All registered individuals will remain in the Registered status until the primary consent is signed.</li>
+              <li><strong>Participant:</strong> A person who has completed the informed consent process and is enrolled in AoURP will become an official Participant.  All Participants will have the opportunity to respond to the HIPAA Authorization/EHR Consent or the DV EHR Intent to Share agreement. If the participant agrees to the sharing EHR data, this person will move to the next status: Fully Consented. If the individual does not agree to share EHR data, the status will remain as a participant. Participants do then have the ability to complete PPI questionnaires and become eligible for PM&B, but will not be considered Core Participants.</li>
```
Comment (shyamnaru) [inline views/dashboard/partials/modal-dashboard-help.html.twig]: Very minor, may be we can use html entity`&amp;` for `&` here and in line 81 as well?

File: views/dashboard/partials/modal-dashboard-help.html.twig (hunk 1 above)
Comment (stephenyeargin) [reply views/dashboard/partials/modal-dashboard-help.html.twig]: 👍

File: views/dashboard/partials/modal-dashboard-help.html.twig (hunk 2 above)
Comment (stephenyeargin) [reply views/dashboard/partials/modal-dashboard-help.html.twig]: 👍

**Pull request merged**